app/views/communicart_mailer/comment_added_email.html.haml

```diff
@@ -0,0 +1,12 @@
+%table.w-container.main-container{width: '800'}
+  - proposal = @comment.proposal
+  %tr
+    %td.w-container.html-email-message
+      %p
+        Action need on Request \##{proposal.public_identifier} (#{proposal.name}):
```

Comment:
I think we can remove the first pound, as the `public_identifier` includes it, hey?